content/docs/user-guide/assets/asset-processor/faster-scanning.md

```diff
@@ -12,16 +12,17 @@ toc: true
 
 ![The Faster Scanning Mode settings in Asset Processor](/images/user-guide/assets/pipeline/asset-processor-interface-fast-scan.png)
 
-Faster Scanning Mode is on by default. To disable Faster Scanning Mode, do the following:
-
+Faster Scanning Mode is on by default for Asset Processor GUI, but off by default for Asset Processor Batch.
+
+To disable Faster Scanning Mode in the GUI, do the following:
 1. Choose the Tools tab in Asset Processor.
 1. Uncheck the **Faster Scanning Mode** option.
 
 {{< note >}}
 When Faster Scanning Mode is disabled, file hashes are used to check for updated source assets and source dependencies. This might significantly increase analysis time.

```

Comment:
I'd remove the "this might significantly increase analysis time" bit, "faster scanning mode" has always been shown to be faster (and the hash algorithm we use is super fast anyway)

Reply:
Fixed